dot/parachain/bitfield-signing/bitfield_signing.go

```diff
@@ -0,0 +1,253 @@
+// Copyright 2024 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package bitfield_signing
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	availabilitystore "github.com/ChainSafe/gossamer/dot/parachain/availability-store"
+	parachaintypes "github.com/ChainSafe/gossamer/dot/parachain/types"
+	parachainutil "github.com/ChainSafe/gossamer/dot/parachain/util"
+	"github.com/ChainSafe/gossamer/internal/log"
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/keystore"
+	"github.com/ChainSafe/gossamer/lib/runtime"
+)
+
+const availabilityDistributionWaitingPeriod = time.Millisecond * 1500
+
+var logger = log.NewFromGlobal(log.AddContext("pkg", "parachain-bitfield-signing"))
```

Comment:
looks like there is inconsistency between subsystems when it comes to the value for `pkg`.

Reply:
shouldn't it be `parachain-subsystem-name`?

Reply:
I'm ok either way, let's discuss with the team.